services/frontend-service/src/ui/components/ReleaseDialog/ReleaseDialog.tsx

```diff
@@ -303,7 +303,11 @@ export const EnvironmentListItem: React.FC<EnvironmentListItemProps> = ({
                             title={
                                 'When doing manual deployments, it is usually best to also lock the app. If you omit the lock, an automatic release train or another person may deploy an unintended version. If you do not want a lock, click the arrow.'
                             }>
-                            <ExpandButton onClickSubmit={deployAndLockClick} defaultButtonLabel={'Deploy & Lock'} />
+                            <ExpandButton
+                                onClickSubmit={deployAndLockClick}
+                                defaultButtonLabel={'Deploy & Lock'}
+                                disabled={release.isPrepublish}
```

Comment:
it's good that the button is disabled.
Please make it a bit more readable. Right now it's black text on dark grey background.

Reply:
done